nodes/villager/Villager.gd

```diff
@@ -84,7 +84,7 @@ const Emotion = {
 
 # a map of emotion to intensity, exported to configure different initial
 # emotions
-export(Dictionary) var exported_emotion_intensity = {
```

Comment:
this exported dict was not playing ball, whenever i created a new villager instance only FEAR and FATIGUE would be populated in this dict, so strange, couldn't figure out why

we can revisit adding this if we want to set villager emotions on spawn